src/Apps/Artist/Components/ArtistHeader/__tests__/ArtistHeader.jest.tsx

```diff
@@ -55,16 +56,22 @@ describe("ArtistHeader", () => {
     expect(screen.getByText("USA, Jan 1 1980")).toBeInTheDocument()
     expect(screen.getByText("111 Followers")).toBeInTheDocument()
     expect(screen.getByText("biographyBlurbText")).toBeInTheDocument()
+    const cvLink = screen.getByText(CV_LINK_TEXT)
```

Comment:
As @mzikherman has flagged this a few times in some of my past tests, wanted to call out that its best not to be too efficient here since we're dealing with unintended breaking changes to code. An unintended change to the variable would still yield a passing test. Maybe he can comment more!

Reply:
Yeah that's what I started with (copy pasted the exact language) but then Jenna has an update to it and that caused me to want to extract a var so that I didn't have to update it all over the place. 🤷

Reply:
Resist the urge @jonallured 😄 

All said tho, minor, and def non-blocking. But it does make the tests more brittle for a tiny gain in efficiency. Do what you will!

Reply:
lol yea, I always advocate _not_ using interpolation or vars in a spec and explicitly write out the expectation. It will have to update there too if the value changes, but seems worth it to keep the spec as clear as possible.

That being said pls don't take my opinion too seriously 😝